google/cloud/bigtable/table.py

```diff
@@ -1078,9 +1081,15 @@ def _do_mutate_retryable_rows(self):
                 client_info=data_client._client_info,
             )
 
-        responses = data_client._inner_api_calls["mutate_rows"](
-            mutate_rows_request, retry=None
-        )
+        try:
+            responses = data_client._inner_api_calls["mutate_rows"](
+                mutate_rows_request, retry=None
+            )
+        except (ServiceUnavailable, DeadlineExceeded, Aborted):
```

Comment:
Can this PR change the `_is_retryable` method on this class to us the imported exceptions, too (to DRY that bit out).

- [Hackage of `grpc.StatusCode`](https://github.com/googleapis/python-bigtable/blob/dcd91c9dceb8cb8dfe89d4e2958e0bde5b413e95/google/cloud/bigtable/table.py#L989-L994)
- [`_is_retryable`](https://github.com/googleapis/python-bigtable/blob/dcd91c9dceb8cb8dfe89d4e2958e0bde5b413e95/google/cloud/bigtable/table.py#L1032-L1034)

Reply:
27b4825